ConfigSpace/conditions.pyx

```diff
@@ -117,8 +117,10 @@ cdef class AbstractCondition(ConditionComponent):
 
     def __richcmp__(self, other: Any, int op):
         """Override the default Equals behavior
-        There are no separate methods for the individual rich comparison operations (__eq__(), __le__(), etc.).
-         Instead there is a single method __richcmp__() which takes an integer indicating which operation is to be performed, as follows:
+        There are no separate methods for the individual rich comparison
```

Comment:
Could you please add a note that is not really necessary any more, but someone would have to change this and link the following website: 
https://cython.readthedocs.io/en/latest/src/userguide/special_methods.html#rich-comparisons